tests/sim_init_tests.cc

```diff
@@ -62,23 +62,23 @@ class Inver : public cy::Facility {
   }
 
   virtual void InitInv(cy::Inventories& inv) {
-    buf1.PushAll(inv["buf1"]);
-    buf2.PushAll(inv["buf2"]);
+    buf1.Push(inv["buf1"]);
+    buf2.Push(inv["buf2"]);
   }
 
   virtual cy::Inventories SnapshotInv() {
     cy::Inventories invs;
     invs["buf1"] = buf1.PopN(buf1.count());
     invs["buf2"] = buf2.PopN(buf2.count());
-    buf1.PushAll(invs["buf1"]);
-    buf2.PushAll(invs["buf2"]);
+    buf1.Push(invs["buf1"]);
+    buf2.Push(invs["buf2"]);
     return invs;
   }
   virtual void Tick() { context()->Snapshot(); }
   virtual void Tock() {};
 
-  cy::toolkit::ResourceBuff buf1;
-  cy::toolkit::ResourceBuff buf2;
+  cy::toolkit::ResBuf<cy::Resource> buf1;
+  cy::toolkit::ResBuf<cy::Resource> buf2;
```

Comment:
Looking at the changes below, it think these should be `Material` buffers
```suggestion
  cy::toolkit::ResBuf<cy::Material> buf1;
  cy::toolkit::ResBuf<cy::Material> buf2;
```

Reply:
Made the switch back to `ResBuf<Material>`.  Required that we use `PopNRes()` at lines 70/71 since `cyclus::Inventories` expects keys of type `cyclus::toolkit::ResVec`